### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated Advanced Image Management

### CODE REVIEW

File: docker-hub/image-management.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **General** and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
+3. In the dashboard you can sory and filter by active, inactive and untagged images.
```
Comment (fdiogo) [inline docker-hub/image-management.md:29]: **typo:** instead of `sory` I believe you meant `sort`

File: docker-hub/image-management.md [hunk 2]
```diff
@@ -43,15 +43,12 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **General** or **Tab** and click Go to Advanced Image Management  from the main menu.
```
Comment (fdiogo) [inline docker-hub/image-management.md:46]: **nitpick:** there seems to be an unnecessary space between `Management` and `from`

**Review by fdiogo — COMMENTED** [review]
The images modified in this PR don't seem to relate to the changes in anyway 🤔 Not sure if I'm doing something wrong though

File: docker-hub/image-management.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:29]: Updated, thanks!

File: docker-hub/image-management.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:46]: Thanks! Updated

File: docker-hub/image-management.md [hunk 3]
```diff
@@ -43,15 +46,12 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **General** or **Tab** and click Go to Advanced Image Management from the main menu.
```
Comment (fdiogo) [inline docker-hub/image-management.md:49]: **typo:** I believe you meant <s>Tab</s> Tags

File: docker-hub/image-management.md [hunk 4]
```diff
@@ -24,11 +24,14 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **General** and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
```
Comment (fdiogo) [inline docker-hub/image-management.md:28]: **suggestion:** we've removed a critical step here which was this one:

> 2. Click **Repositories** from the main menu and select a repository from the list

Without it, the current step list doesn't make much sense 🤔 This is still needed before we go to the **General** tab, since it only exists within a repository

File: docker-hub/image-management.md (hunk 3 above)
Comment (fdiogo) [inline docker-hub/image-management.md:49]: Same comment around removing the critical step of going to the repositories first

File: docker-hub/image-management.md (hunk 4 above)
Comment (fdiogo) [inline docker-hub/image-management.md:28]: **suggestion:** I think this phrasing could be changed to match the one below, since it's more complete

> 2. Click **General** or **Tab** and click Go to Advanced Image Management from the main menu.

File: docker-hub/image-management.md [hunk 5]
```diff
@@ -43,15 +46,12 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **General** or **Tab** and click Go to Advanced Image Management from the main menu.
+3. Click **Inactive images** and select the images that you want to delete from the repository and click the delete icon.
```
Comment (fdiogo) [inline docker-hub/image-management.md:50]: **suggestion:** There's no "Inactive images" button or link now, that was previously located on a banner that no longer exists.

We could rephrase this to something like:
> 3. Enable the "Inactive" filter to view your inactive images.

> 4. Select the images that you want to delete from the repository and click the delete icon.

File: docker-hub/image-management.md [hunk 6]
```diff
@@ -43,15 +46,12 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
...
 
     ![Advanced image management dashboard](images/dashboard-delete-image.png){:width="700px"}
 
-6. Carefully read the advice displayed about deleting the image you have selected.
-7. Select the check box to confirm your action and then click **Delete**.
+4. Select the check box to confirm your action and then click **Delete**.
```
Comment (fdiogo) [inline docker-hub/image-management.md:54]: **suggestion:** remove this step as it's repeating the previous step

File: docker-hub/image-management.md [hunk 7]
```diff
@@ -24,11 +24,14 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **General** and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
+
+![Advanced image management dashboard](images/advanced-image-general.png){:width="700px"}
```
Comment (fdiogo) [inline docker-hub/image-management.md:30]: **suggestion:** we could add an image of the Tags tab as well since there's a link to navigate to the advanced image management there

**Review by fdiogo — CHANGES_REQUESTED** [review]

File: docker-hub/image-management.md (hunk 4 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:28]: Updated

Comment (jerae-duffin) [reply docker-hub/image-management.md:28]: Updated

File: docker-hub/image-management.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:49]: Updated

Comment (jerae-duffin) [reply docker-hub/image-management.md:49]: updated

File: docker-hub/image-management.md (hunk 5 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:50]: Updated

File: docker-hub/image-management.md [hunk 8]
```diff
@@ -24,11 +24,15 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tag**, and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
```
Comment (fdiogo) [inline docker-hub/image-management.md:29]: **suggestion:** the tab says **Tags** (in plural), here it's stated in the singular form

File: docker-hub/image-management.md [hunk 9]
```diff
@@ -43,15 +47,13 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tag** and click Go to Advanced Image Management from the main menu.
```
Comment (fdiogo) [inline docker-hub/image-management.md:51]: same comment around **Tag** being in singular form while the tab has it in plural

Comment (jerae-duffin) [reply docker-hub/image-management.md]: Thanks for catching this @fdiogo, updated!

File: docker-hub/image-management.md [hunk 10]
```diff
@@ -24,11 +24,15 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
 
 To access the  Advanced Image Management dashboard:
 
-1. Log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard. It lists images in your repository and allows you to sort and filter images by tags, activity status, and the last activity date.
+1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tags**, and click **Go to Advanced Image Management** from the main menu, and select a repository from the list.
```
Comment (fdiogo) [inline docker-hub/image-management.md:29]: **suggestion:** the `Go to Advanced Image Management` links are not in the main menu, and you can't select a repository when you're there, since you're already on a repository when you enter the image management dashboard

File: docker-hub/image-management.md [hunk 11]
```diff
@@ -24,11 +24,15 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
...
+
+![Advanced image management dashboard](images/advanced-image-general.png){:width="700px"}
```
Comment (fdiogo) [inline docker-hub/image-management.md:31]: **suggestion:** could we swap this image with the newer one I sent with the `mobythewhale` user?

File: docker-hub/image-management.md [hunk 12]
```diff
@@ -43,15 +47,13 @@ The Advanced Image Management dashboard provides a snapshot of your existing ima
 By default, the Advanced Image Management dashboard displays inactive images in the repository. To delete an inactive image:
 
 1. Log in to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"}.
-2. Click **Repositories** from the main menu and select a repository from the list. This displays a banner indicating the number of inactive images in the repository.
-3. Click **View inactive images**. This displays the Advanced Image Management dashboard.
-4. Select the images that you’d like to delete from the repository.
-5. Click the delete icon located above the list of images.
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tags** and click Go to Advanced Image Management from the main menu.
```
[the remark this answers is not in the document]
Comment (fdiogo) [reply docker-hub/image-management.md:51]: **nitpick:** the formatting here is different than the same step on top of this file

**Review by fdiogo — COMMENTED** [review]
Left a few more comments around the wording and the images used 😃 

Was wondering if we could add the images that show the `Go to Advanced Image Management` links on the `Tags` tab as well?

File: docker-hub/image-management.md (hunk 10 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:29]: @fdiogo Step 2 is correct, but step 3 should I  remove it completely?

File: docker-hub/image-management.md (hunk 11 above)
Comment (jerae-duffin) [reply docker-hub/image-management.md:31]: Updated in new commit

File: docker-hub/image-management.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/image-management.md:51]: Updated in new commit. Thank you!

File: docker-hub/image-management.md [hunk 13]
```diff
@@ -24,11 +24,17 @@ With the new Advanced Image Management dashboard, all of this is now simpler and
...
+2. Click **Repositories** from the main menu and select a repository from the list.
+3. Click **General** or **Tags**, and click **Go to Advanced Image Management**.
+
+  ![Advanced image management dashboard](images/advanced-image-general.png){:width="700px"}
+
+4. In the dashboard you can sort and filter by active, inactive and untagged images.
 
-    > You must be logged in as a Docker Pro, Team, or a Business user to use the Advanced Image Management dashboard. Click [Upgrade](https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade){: target="_blank" rel="noopener" class="_"} to upgrade your existing subscription.
+    > **Note**
+    >
```
Comment (fdiogo) [inline docker-hub/image-management.md:36]: **nitpick:** I think there's a leftover quote here

**Review by craig-osterhout — APPROVED** [review]
LGTM
